aerleon/lib/nacaddr.py

```diff
@@ -516,12 +492,5 @@ def AddressListExclude(
         return sorted(set(ret_array + superset))
 
 
-ExcludeAddrs = AddressListExclude
```

Comment:
Seems like third party generators could still be using ExcludeAddrs and Supernet, especially if they looked at code in the built-in generators using them. A deprecation notice seems warranted.

Reply:
I should say, a deprecation notice would be warranted and we should not remove ExcludeAddrs and Supernet right away.